fml/src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -586,14 +590,20 @@ public void firePlayerChangedDimensionEvent(EntityPlayer player, int fromDim, in
         bus().post(new PlayerEvent.PlayerChangedDimensionEvent(player, fromDim, toDim));
     }
 
-    public void firePlayerLoggedIn(EntityPlayer player)
+    public IChatComponent firePlayerLoggedIn(EntityPlayer player, IChatComponent loginMessage)
```

Comment:
Why would this return an IChatComponent?

Shouldn't it be like the logout event, where it checks the message and then sends it to the players?